package.json

```diff
@@ -3,7 +3,7 @@
 	"private": true,
 	"license": "Apache-2.0",
 	"scripts": {
-		"test": "turbo run test",
+		"test": "turbo run test --env-mode=loose",
```

Comment:
This env mode kind of scares me, is there a way to just manually define the config and use the prefixed env vars?

Reply:
yeah, i can create a `turbo.json` and add the specific env vars required for aws kms signers
https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#loose-mode